server/statistics/hot_peer_cache.go

```diff
@@ -456,14 +454,15 @@ func (f *hotPeerCache) updateHotPeerStat(region *core.RegionInfo, newItem, oldIt
 		if newItem.inCold {
 			f.coldItem(newItem, oldItem)
 		} else {
+			thresholds := f.calcHotThresholds(newItem.StoreID)
 			if f.isOldColdPeer(oldItem, newItem.StoreID) {
```

Comment:
The judgement is too difficult，maybe need to be refactored  it .

Reply:
It is not the content of this pr